lib/Backends/Interpreter/InterpreterDeviceManager.cpp

```diff
@@ -51,6 +51,17 @@ bool InterpreterDeviceManager::isMemoryAvailable(uint64_t estimate) const {
   return maxMemoryBytes_ >= (usedMemoryBytes_ + estimate);
 }
 
+DeviceInfo InterpreterDeviceManager::getDeviceInfo() const {
+  // TODO: these may need to be tweaked depending on specific CPU
```

Comment:
Could you please explain more about "specific CPU" here?  It is Interpreter backend.